packages/client/src/containers/Login.container/SignupForm.js

```diff
@@ -10,24 +10,16 @@ const SignupForm = () => {
   });
   const [formErrors, setFormErrors] = useState({});
   const [isSubmit, setIsSubmit] = useState(false);
-  const message = '';
 
   const handleChange = (e) => {
     setFormValues({ ...formValues, [e.target.name]: e.target.value });
   };
 
   const handleSubmit = (event) => {
```

Comment:
I meant that:
```
const handleSubmit = (event) => {
 event.preventDefault();
 if(Object.keys(formErrors).length === 0) {
 setIsSubmit(true);
 }
  }; 
```
Validations are not supposed to be made on submit time, but after the user types the input. Usually, we use the onBlur propertie to run a validation function and then reset it onFocus.

Reply:
@annamourasobral  okay Thanks, 
I need to add the form validation as well , right?
```
const handleSubmit = (event) => {
 event.preventDefault();
**setFormErrors(FormValidation(formValues));**
 if(Object.keys(formErrors).length === 0) {
 setIsSubmit(true);
 }
  }; 
```